integration-tests/testutils/report/parser.go

```diff
@@ -95,12 +95,12 @@ func (fr *SubunitV2ParserReporter) Write(data []byte) (int, error) {
 
 	if matches := announceRegexp.FindStringSubmatch(sdata); len(matches) == 2 {
 		testID := matches[1]
-		if isTest(testID) {
+		if isTest(testID) && !common.IsInRebootProcess() {
 			err = fr.statuser.Status(subunit.Event{TestID: matches[1], Status: "exists"})
 		}
 	} else if matches := successRegexp.FindStringSubmatch(sdata); len(matches) == 2 {
```

Comment:
Sure thing. This is clearly a hack because we currently can't extend the results reporter of gocheck. Do you remember I asked you about a couple of pull requests related to this in the gocheck github repo? Once we get the tests running in the cloud and the lab for merge proposals, we can go back to this.

About this specific fragility, it's not critical. ATM this is just to have a pretty print of the test results, but they will keep failing even if we can't generate the subunit, and the full test log will be available to check what failed. This will be more important once we start to run things in parallel and use subunit more extensively.